base/deprecated.jl

```diff
@@ -1853,6 +1853,12 @@ end
         callback(r)
     end
     nothing
+
```

Comment:
Bad rebase, missing `end`

Reply:
Good catch, thanks. The merging algorithm doesn't work very well in this kind of situation. :-)